src/nested_formset/tests/test_factory.py

```diff
@@ -60,3 +61,61 @@ def test_exclude_fields_for_factory(self):
             tuple(nested_formset.form.base_fields.keys()),
             ('block',),
         )
+        
+    def test_fk_name_for_factory(self):
+        
+        fk_name = 'block'
+        # Should pass because fk_name is valid
+        nested_formset = nestedformset_factory(
+            test_models.Block,
+            test_models.Building,
+            nested_formset=self.child_formset,
+            fk_name='block'
+        )()
+        # Fails because address is not fk 
+        with self.assertRaises(ValueError):
+            nested_formset = nestedformset_factory(
+                test_models.Block,
+                test_models.Building,
+                nested_formset=self.child_formset,
+                fk_name='address'
+            )()
+        
+    def test_min_num_for_factory(self):
+        
+        num = 3
+        nested_formset = nestedformset_factory(
+            test_models.Block,
+            test_models.Building,
+            nested_formset=self.child_formset,
+            min_num=num
+        )
+        
+        if django.VERSION >= (1, 7):
```

Comment:
Django 1.7 is [no longer supported](https://www.djangoproject.com/download/#supported-versions), so I think you can drop conditional here and below.